### ISSUE TITLE
Unexpected line chart behaviour

### ISSUE THREAD

**Issue by ralight** [MeasureOSS/Measure#92]
Total open PRs, earlier this evening:
![before](https://user-images.githubusercontent.com/79578/38648830-347f557a-3deb-11e8-804b-cb553844fae2.png)
And after I've closed some PRs:
![after](https://user-images.githubusercontent.com/79578/38648838-3e89346e-3deb-11e8-89be-8f5f6ac574e4.png)

The whole right hand side of the graph has shifted downwards. My understanding is that the line chart is showing me the number of open pull requests I had in a given month. If that's the case, closing PRs now shouldn't affect the results for last month. Same thing is happening with the issues line chart.

**Comment by ralight** [comment]
There is related behaviour on the PRs open and closed this month which makes me suspect it is my expectation of what the graph is showing is wrong.

I think:

Open/close state for a given month

Looks like:

How many were closed in a given month, also how many are currently still open from that month.

That kind of makes sense for the bar charts, you have a log of your activity (closed) plus an indicator of the backlog and how many old PRs there are. A bit confusing without some explanation though.

**Comment by stuartlangridge** [comment]
Yup. The issue is that knowing how many were opened in a given month isn't really as helpful as knowing how many are still open, if you see what I mean. The goal is to do the intuitive thing rather than necessarily the mathematically correct thing. This may need some tweaking, of course.
